### PR TITLE
Remove extraneous word

### PR DESCRIPTION
**Pull request by TerryHowe**

### CODE REVIEW

File: docs/contents/community/faq.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## A Kubernetes distribution?
 
 !!! question "Question"
-    What is a Kubernetes distribution and why do you call EKS Distro one?
+    What is a Kubernetes distribution and why do you call EKS Distro?
```
Comment (rothgar) [inline docs/contents/community/faq.md:6]: This sentence doesn't make grammatical sense to me anymore. Maybe
>Why do you call EKS Distro a Kubernetes distribution?

Comment (abhay-krishna) [reply docs/contents/community/faq.md:6]: Or
> ... and how does EKS Distro classify as one?

Comment (TerryHowe) [inline docs/contents/community/faq.md:6]:
```suggestion
    Why do you call EKS Distro a Kubernetes distribution?
```

Comment (TerryHowe) [reply docs/contents/community/faq.md]: Done

**Review by jaxesn — APPROVED** [review]
/lgtm

**Pull request merged**